src/test/compile-fail/issue-23024.rs

```diff
@@ -18,6 +18,7 @@ fn main()
     vfnfer.push(box h);
     println!("{:?}",(vfnfer[0] as Fn)(3));
     //~^ ERROR the precise format of `Fn`-family traits'
-    //~| ERROR wrong number of type arguments: expected 1, found 0
+    //~| ERROR E0243
+    //~| NOTE expected 1 type arguments, found 0
```

Comment:
Right.  For the time being, would you mind adding the 4 missing ones?  We'll probably do another pass later on that helps clean up macro errors, but for now we'll need to test for them.